rke/k8s_rke_system_images.go

```diff
@@ -5655,10 +5655,10 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 		"v1.19.16-rancher1-3": {
 			Etcd:                      "rancher/mirrored-coreos-etcd:v3.4.15-rancher1",
 			Kubernetes:                "rancher/hyperkube:v1.19.16-rancher1",
-			Alpine:                    "rancher/rke-tools:v0.1.79",
-			NginxProxy:                "rancher/rke-tools:v0.1.79",
-			CertDownloader:            "rancher/rke-tools:v0.1.79",
-			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.79",
+			Alpine:                    "rancher/rke-tools:v0.1.78",
```

Comment:
You're only updating the latest mirror version of kubernetes right, hyperkube update 3? I want to make sure I understand why rancher1-1 and 1-2 are left out